package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "cht-upgrade-service",
```

Comment:
I'm not sure if it's worth publishing it. 
Right now the paths to the docker-compose folder location are hardcoded, because we control the volumes mounted in config. We'd have to account for use outside of docker, passing file location differently. Also e2e tested when it's not running in a container. 

It's zero cost to add `@medic`, on the other hand.